near/nep141-locker/src/storage.rs

```diff
@@ -0,0 +1,142 @@
+use near_contract_standards::storage_management::{StorageBalance, StorageBalanceBounds};
+use near_sdk::{assert_one_yocto, borsh};
+use near_sdk::{env, near_bindgen, AccountId, NearToken};
+
+use crate::*;
+
+pub type TransferMessageStorageValue = (TransferMessage, AccountId);
```

Comment:
Maybe it's better to add named fields. Right now, it's not clear what AccountId means.

Reply:
The goal was to avoid extra clones, but yeh, it's not worth it